src/taxes/pages/TaxChannelsPage/useTaxStrategyChoices.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+  TaxCalculationStrategy,
+  useTaxStrategyChoicesQuery,
+} from "@dashboard/graphql";
+import React from "react";
+
+import { FlatTaxRateLabel, TaxAppLabel } from "../../components";
+
+const flatTaxRateChoice = {
+  label: <FlatTaxRateLabel />,
+  value: TaxCalculationStrategy.FLAT_RATES,
+};
+
+export const useTaxStrategyChoices = () => {
+  const { data, loading } = useTaxStrategyChoicesQuery();
+  const taxAppsChoices =
+    data?.shop.availableTaxApps.map(app => ({
+      value: app.identifier,
+      label: (
+        <TaxAppLabel
+          name={app.name}
+          created={app.created}
+          version={app.version}
+          id={app.id}
+          identifier={app.identifier}
+          logoUrl={app.brand?.logo?.default}
+        />
+      ),
+    })) ?? [];
+
+  const choices = [...taxAppsChoices, flatTaxRateChoice];
+  return [choices, loading] as const;
```

Comment:
I think this API is worse than `return {choices, loading} as const`.

As a user of this hook I need to check its internal code to understand what is behind the tuple. **And I will name them "choices" and "loading" anyway**

This is not react "useState" → here we are specific what is behind the values